forge-gui/res/cardsfolder/upcoming/the_temporal_anchor.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Temporal Anchor
+ManaCost:3 U U U
+Types:Legendary Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigScry | TriggerDescription$ At the beginning of your upkeep, scry 2.
+SVar:TrigScry:DB$ Scry | ScryNum$ 2
+T:Mode$ Scry | ValidPlayer$ You | ToBottom$ True | Execute$ TrigExile | TriggerDescription$ Whenever you choose to put one or more cards on the bottom of your library while scrying, exile that many cards from the bottom of your library.
+SVar:TrigExile:DB$ Dig | DigNum$ X | ChangeNum$ All | FromBottom$ True | DestinationZone$ Exile
+SVar:X:TriggerCount$ScryBottom
+S:Mode$ Continuous | Condition$ PlayerTurn | MayPlay$ True | Affected$ Card.ExiledWithSource | AffectedZone$ Exile | Description$ During your turn, you may play cards exiled with CARDNAME.
```

Comment:
The problem is unless ExiledWithSource would be fixed with a timestamp check you can flicker the Anchor and still play the old cards

Reply:
For now, you need to remember the exiled cards

Reply:
Good call – thanks, you two